tla/consensus/Traceccfraft.cfg

```diff
@@ -20,7 +20,7 @@ POSTCONDITION
 \* matches the full log.  However, TLA cannot check this property directly because it
 \* is a hyperproperty.
 PROPERTIES
-    CCFSpec
+    \* CCFSpec
```

Comment:
@lemmy: I've disabled this refinement check to debug the trace validation in isolation. I believe that's now passing for all current scenarios, but fails with the refinement check. Can you take a look at the TLA changes I've made (removed `DropMessages` from `TraceSpec`), and explain what we need to do to restore refinement checks?

Reply:
Update: I think I've resolved this, and have now restored the refinement check and removed `DropMessages` from `Traceccfraft`. Please vet this change!

Reply:
LGTM